test/integration/enable/test_enable.py

```diff
@@ -0,0 +1,177 @@
+import os
+import re
+import shutil
+import subprocess
+
+import yaml
+
+
+def test_enable_application(tt_cmd, tmpdir):
+    test_app_path_src = os.path.join(os.path.dirname(__file__), "test_app")
+    config_path = os.path.join(tmpdir, "tt.yaml")
+    instances_enabled_path = os.path.join(tmpdir, "bar")
+    with open(config_path, "w") as f:
+        yaml.dump({"env": {"instances_enabled": instances_enabled_path}}, f)
+    os.makedirs(instances_enabled_path)
+    test_app_path = os.path.join(tmpdir, "test_app")
+    shutil.copytree(test_app_path_src, test_app_path)
+
+    # Enable test application.
+    enable_cmd = [tt_cmd, "enable", test_app_path]
+    enable_process = subprocess.Popen(
+        enable_cmd,
+        cwd=tmpdir,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        text=True
+    )
+    enable_process.wait()
+
+    # Check that application is enabled.
+    instances_cmd = [tt_cmd, "instances"]
+    instance_process = subprocess.Popen(
+        instances_cmd,
+        cwd=tmpdir,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        text=True
+    )
+    start_output = instance_process.stdout.read()
+    assert re.search("test_app", start_output)
+
+
+def test_enable_script(tt_cmd, tmpdir):
+    test_script_path_src = os.path.join(os.path.dirname(__file__), "test_script", "test.lua")
+    config_path = os.path.join(tmpdir, "tt.yaml")
+    instances_enabled_path = os.path.join(tmpdir, "bar")
+    with open(config_path, "w") as f:
+        yaml.dump({"env": {"instances_enabled": instances_enabled_path}}, f)
```

Comment:
Please, check that enable will not work in case of instances_enabled = .

Reply:
added test